src/lib/__tests__/cache.test.js

```diff
@@ -38,6 +38,17 @@ describe("Cache", () => {
       })
     })
 
+    describe("#delete", () => {
+      beforeEach(() => cache.set("get_foo", { bar: "baz" }))
+
+      it("deletes the data", () => {
+        cache.delete("get_foo")
+        return cache.get("get_foo").catch(e => {
+          expect(e.message).toEqual("cache#get did not return `data`")
```

Comment:
Thanks for this!

Does the client really raise an exception when getting an unset value?

Reply:
Yup! It rejects the promise with an error, so I believe we are handling it correctly. That was the old behavior as well.